server.js

```diff
@@ -2094,7 +2094,7 @@ cache(function(data, match, sendBadge, request) {
     }
     try {
       var parsedData = JSON.parse(buffer);
-      if (info.charAt(0) === 'd') {
+      if ((info === 'dm') || (info === 'dw') || (info ==='dd')) {
```

Comment:
You can omit the parens around these, since `||` binds at a lower precedence. Everywhere but `server.js` I think the linter will catch it.